src/goto-symex/symex_builtin_functions.cpp

```diff
@@ -10,6 +10,7 @@ Author: Daniel Kroening, kroening@kroening.com
 /// Symbolic Execution of ANSI-C
 
 #include "goto_symex.h"
+#include <iostream>
```

Comment:
Is this left over from some debugging?

Reply:
Actually newly added debugging: one test is failing on Windows, and I don't really understand why that's the case.